src/vs/workbench/browser/parts/views/panelViewlet.ts

```diff
@@ -119,8 +119,8 @@ export abstract class ViewletPanel extends Panel implements IView {
 		this.updateActionsVisibility();
 	}
 
-	protected renderHeaderTitle(container: HTMLElement): void {
-		append(container, $('.title', null, this.title));
+	protected renderHeaderTitle(container: HTMLElement, classList: string[], children: (string | Node)[]): void {
```

Comment:
Passing class names and children to render header title looks weird to me. It is also not sure what this method doing. I would rather have this method just rendering the `title` given the container element and the title string. 

`renderHeaderTitle(container: HTMLElement, title: string)`

Subclasses can decide when to render the tile of their views by just passing the container and the title.